locales/th-TH/translation.json

```
@@ -92,65 +92,65 @@
         "current-day": "ไปที่วันปัจจุบัน",
         "previous-day": "เมื่อวานนี้",
         "next-day": "พรุ่งนี้",
-        "time-to-leave": "Time to Leave",
+        "time-to-leave": "เวลาเลิกงาน",
```

Time to Leave is the name of the App, because of this we are not translating it. :)
```suggestion
        "time-to-leave": "Time to Leave",
```